docs/MODULES.md

```diff
@@ -68,20 +71,90 @@ For more information please see the [CJDNS FAQ](https://github.com/cjdelisle/cjd
 
 To modify the ports that are accessable from CJDNS modify the `cjdns` *table* in the IPv6 firewall config file. See the [**Firewall**](#firewall) section for more details.
 
+### CJDNS Internet Peering
+
+Other peers can be found in the [this](https://github.com/hyperboria/peers) repo of peers. Try to connect to only a few peers, and ones that are close to where you live.
+
+You'll see the peering information that will give you the address (ipv4 or ipv6) and credentials to connect to the node. You must either select to use just the ipv4 config, or you could use both. Now that we have this info, connect to your mesh device, and edit the following config file:
+
+```
+$ sudo nano /etc/cjdroute.conf
+```
+
+This file contains everything that is required for CJDNS to run, so be careful not to remove anything else, unless you know what you are doing. You need to head down to this line:
+
+```
+// Nodes to connect to (IPv4 only).
+"connectTo":
+{
+```
+
+This is where you input the ipv4 address. There is also a ipv6 field:
+
+```
+// Nodes to connect to (IPv6 only).
+"connectTo":
+{
+```
+
+Insert the respective code, and the save (ctrl+X to save, then ctrl+S to confirm file name, then ENTER to confirm changes).
+your code should look somewhat like this:
+```
+// Nodes to connect to (IPv4 only).
+"connectTo":
+  {
+                "123.123.123.123:54321": {
+                  "peerName": "cosmetic-name.com",
+                  "login": "peer-login",
+                  "password": "peer-passwowrd-here",
+                  "publicKey": "1234567890123456789012345678901234567890123456789012.k"
+                }
+  }
+```
+
+Next you should restart CJDNS with a `sudo systemctl restart cjdns` command. This will reload CJDNS with the new config file. Run a `status` command on your node, and make sure when it prints out
```

Comment:
```suggestion
Next you should restart CJDNS with the `sudo systemctl restart cjdns` command. This will reload CJDNS with the new config file. Run the `status` command on your node, and make sure when it prints out
```